# Generated by Django 3.2.6 on 2021-10-21 18:39

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('lostandfound', '0005_lostobject'),
    ]

    operations = [
        migrations.CreateModel(
            name='FoundObject',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('obj_item_found', models.CharField(max_length=50)),
                ('obj_date_found', models.DateField()),
                ('obj_location_found', models.CharField(max_length=50)),
                ('obj_description_found', models.CharField(max_length=100)),
                ('obj_picture_found', models.ImageField(blank=True, null=True, upload_to='media', verbose_name='Image')),
                ('obj_number_found', models.CharField(max_length=50)),
                ('o_post_date_found', models.DateTimeField(default=django.utils.timezone.now)),
                ('uploader', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
